include/atomicassets.hpp

```diff
@@ -109,7 +109,13 @@ CONTRACT atomicassets : public contract {
         name collection_name,
         int32_t template_id
     );
-
+    
+    ACTION redmaxtemplt(
```

Comment:
This is just a nitpick, but I would prefer a different name. `createtempl` abbreviates template as templ, so redmaxtemplt would read red(uce)-max-templ(ate)-t. I assume the idea is that it means red(uce)-temp(late)-l(imi)t, but it's inconsistent with how template was abbreviated previously.

I would suggest something like `redtemplmax`